identity-chaining-draft.md

```diff
@@ -6,7 +6,7 @@ submissiontype: IETF
 area: sec
 wg: oauth
 
-docname: draft-ietf-oauth-identity-chaining-00
+docname: draft-ietf-oauth-identity-chaining-01
```

Comment:
this should actually be `-latest` and the build tool will do the right thing

Reply:
actually, this PR predates the move to that build tool and should probably just be closed with a new PR to fix the id nits non-asciii per 82a4600efce97500f5b128e46e7c0e6a42ab0b4d